content/docs/sidebar.json

```
@@ -100,8 +100,11 @@
         "slug": "how-to",
         "source": false,
         "children": [
-          "add-output-to-stage",
           "undo-adding-data",
+          {
+            "label": "Add Deps or Outs to a Stage",
+            "slug": "add-deps-or-outs-to-a-stage"
+          },
```

Actually, I guess we can just use the slug then, like Ivan suggested:
```suggestion
          "add-deps-or-outs-to-a-stage",
```

---

Committing...